optimizely/helpers/validator.py

```diff
@@ -14,6 +14,7 @@
 import json
 import jsonschema
 
+from six import string_types
```

Comment:
nit. This is an external package so it should be imported on line 16 and line 17 should be empty.